charts/nx-agents/templates/daemonset.yaml

```diff
@@ -0,0 +1,43 @@
+{{- if .Values.daemonset.enabled }}
```

Comment:
this is for the inotify watchers right? Do we know what issues we ran into and why we need them? Will Enterprises need them as well?